pkg/pqm4/Makefile

```diff
@@ -0,0 +1,12 @@
+PKG_NAME=pqm4
+PKG_URL=https://github.com/jowlo/pqm4.git
+PKG_VERSION=riot-port
+PKG_LICENSE=PD
+
+.PHONY: all
+
+all: git-download
```

Comment:
Please include a warning message on the state of the algorithm that is printed when you build this package.